txtdirect.go

```diff
@@ -124,27 +128,31 @@ func isIP(host string) bool {
 	return err == nil
 }
 
-// Redirect the request depending on the redirect record found
-func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
-	w.Header().Set("Server", "TXTDirect")
-
-	host := r.Host
-	path := r.URL.Path
-
-	bl := make(map[string]bool)
-	bl["/favicon.ico"] = true
-
-	if bl[path] {
-		redirect := strings.Join([]string{host, path}, "")
+func blacklist(w http.ResponseWriter, r *http.Request, c Config) error {
```

Comment:
Nit: Not sure blacklist does describe the function well. Maybe blacklistRedirect? Not sure.

Reply:
Can't think of anything better than `blacklistRedirect`. It's clear enough and I'm fine with it.